src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -165,6 +171,15 @@ public TopicPartitionWriter(
       this.hiveUpdateFutures = hiveUpdateFutures;
       hivePartitions = new HashSet<>();
     }
+
+    String timeZoneString = (String) connectorConfig.getString(HdfsSinkConnectorConfig.TIMEZONE_CONFIG);
```

Comment:
Agreed on checking only only of scheduled rotation enabled. I want to keep timezone explicit to avoid errors during environment changes